.eslintrc.js

```diff
@@ -43,6 +43,16 @@ module.exports = {
         ],
       },
     },
+    {
+      files: ['scripts/*.ts'],
+      rules: {
+        // Even though the root package is unpublished, we still want to import
+        // published packages.
```

Comment:
Hmm. This doesn't seem right. This rule is about ensuring you don't use packages not listed in the manifest, and that you don't use devDependencies in files that are shipped with the package. It doesn't prevent importing published things.

Reply:
Oh, I think I figured it out. We should leave this rule on but add an empty `files` property to the root manifest. That will stop this rule for mistaking the script as something we publish.

Reply:
I tried that but ESLint seems to strip it. My guess is that since the package is `private`, it doesn't see a need for `files`. But not sure where that is happening yet.

Reply:
Hmm, it worked for me. Did you delete the `.eslintcache` file when you tried? I noticed it failed until clearing the cache, then it worked.

Reply:
Ah, I realize this is coming from my Yarn constraints rules 🤦🏻 Fixed here: [9c67258](https://github.com/MetaMask/controllers/pull/976/commits/9c67258ba86a18fc24463a0786ecf71854d28e73)